app/package.json

```diff
@@ -24,19 +42,26 @@
     "parcel-bundler": "1.12.3",
     "popper.js": "1.15.0",
     "prettier": "1.18.2",
-    "react": "16.8.0",
-    "react-bootstrap": "1.0.0-beta.11",
-    "react-dom": "16.8.0"
+    "react": ">=16.8.0",
+    "react-bootstrap": "^1.0.0-beta.11",
+    "react-dom": ">=16.8.0",
+    "testing-library": "^0.0.2"
   },
   "devDependencies": {
     "@babel/core": "7.0.0",
-    "@babel/preset-react": "7.6.3"
+    "@babel/preset-react": "7.6.3",
+    "@testing-library/jest-dom": "^4.2.3",
```

Comment:
please make sure to pin these versions. i'm also guessing you're manually adding these to the file. you should ideally be using:
```
$ yarn add <package_name> -E
```